src/Microsoft.DotNet.Darc/src/Darc/Helpers/UxManager.cs

```diff
@@ -37,9 +37,9 @@ public int PopUp(EditorPopUp popUp)
                 return Constants.ErrorCode;
             }
 
-            if (string.IsNullOrEmpty(_gitDir))
+            if (string.IsNullOrEmpty(_rootDir))
```

Comment:
>_rootDir [](start = 37, length = 8)

If we are doing `string path = Path.Combine(Path.GetTempPath(), Path.GetTempFileName(), popUp.Path);` then _rootDir is not needed here

Reply:
Removed.

---
In reply to: [264385043](https://github.com/dotnet/arcade-services/pull/206#discussion_r264385043) [](ancestors = 264385043)